FTL.h

```diff
@@ -283,7 +283,6 @@ extern bool threadreadlock;
 extern unsigned char blockingstatus;
 
 extern char * username;
-extern char timestamp[16];
```

Comment:
This global variable is never declared so this `extern ...` definition should not exist.